app/code/Magento/MediaGalleryApi/Api/IsPathBlacklistedInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MediaGalleryApi\Api;
+
+/**
+ * Directory paths that are reserved by system and not be included in the media gallery
```

Comment:
Clarify descriptions

Reply:
What is not clear?

Reply:
Check if the path is blacklisted for media gallery. Directory path may be blacklisted if it's reserved by the system